.vscode/settings.json

```diff
@@ -0,0 +1,45 @@
+{
```

Comment:
Just one comment, be careful with vscode configuration

Reply:
Thanks for spotting that :+1: